build/monitor.js

```diff
@@ -29,6 +29,14 @@ module.exports = function (options) {
 
   var trackers = [];
 
+  if (options.http) {
+    var HttpValidationTracker = require('./http/validation_tracker');
+    if (!process.env.HTTP_SERVICE_URL) {
+      return console.log('HTTP_SERVICE_URL required');
```

Comment:
Oh I see the problem, you overrode the `-h` flag which previously was an alias for `--help`